### PR TITLE
[4.0] Dashboard links

### PR DESCRIPTION
**Pull request by brianteeman**
The text of the new dashboard links is currently "Dashboard: Content"

It is better english to say "Content Dashboard".

This PR also removes the :

When a screen reader sees a : it inserts a long pause and as this text is really just for screen readers it makes it confusing

### CODE REVIEW

File: administrator/modules/mod_menu/tmpl/default_submenu.php [hunk 1]
```diff
@@ -127,7 +127,7 @@
 
 if ($current->dashboard)
 {
-	$titleDashboard = Text::_('MOD_MENU_DASHBOARD_LINK') . ': ' . Text::_($current->title);
+	$titleDashboard = Text::_($current->title) . ' ' . Text::_('MOD_MENU_DASHBOARD_LINK');
```
Comment (wilsonge) [inline administrator/modules/mod_menu/tmpl/default_submenu.php]: As we're doing this can we make this a sprintf please for when we start doing multilang

Comment (brianteeman) [reply administrator/modules/mod_menu/tmpl/default_submenu.php]: Sure thing

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Comment by infograf768** [comment]
Has this been tested in RTL?

**Comment by infograf768** [comment]
Forget it. I see it was modified to use a sprintf.

**Pull request merged**